rllib/models/catalog.py

```diff
@@ -104,6 +105,29 @@ class ModelCatalog:
         >>> action = dist.sample()
     """
 
+    @staticmethod
+    def _make_bounded_dist(action_space):
```

Comment:
Note this doesn't work if you have a tuple action space containing a `DiagGaussian` with more than 1 dim where at least one of them is bounded. This produces nested `MultiActionDistribution`s, which I don't think are supported at the moment.  Is this the case?

Reply:
Not sure actually what would happen. Generating the MultiActionDistribution in catalog.py would work even for nested Tuple spaces. Just leave it as is for now. We'll clean this up very soon in our ongoing torch-ft unification attempt. I would like to make any space (action and observation) work, but lots of changes are needed for that in the different Algos (mostly in the loss functions to reduce log-probs/entropies) to support that.

Reply:
Also, I don't think there is a need right now for Boxes with different bounds on the different axes. So should be totally fine for now.